### PR TITLE
NR-707 - golden metrics in observability panel

### PR DESCRIPTION
**Pull request by eric-j**

### CODE REVIEW

File: shared/agent/src/providers/newrelic.ts [hunk 1]
```diff
@@ -2189,7 +2191,7 @@ export class NewRelicProvider extends ThirdPartyIssueProviderBase<CSNewRelicProv
 			return {
 				goldenMetrics: goldenMetrics,
 				newRelicEntityAccounts: observabilityRepo.entityAccounts,
-				newRelicAlertSeverity: entity.alertSeverity,
+				newRelicAlertSeverity: entity.alertSeverity || "hi",
```
Comment (bcanzanella) [inline shared/agent/src/providers/newrelic.ts:2194]: "hi"

File: shared/ui/Stream/Observability.tsx [hunk 2]
```diff
@@ -739,111 +735,122 @@ export const Observability = React.memo((props: Props) => {
...
+																				name="link-external"
+																				className="clickable"
+																				title="View on New Relic"
+																				placement="bottomLeft"
+																				delay={1}
+																				onClick={e => {
+																					e.preventDefault();
+																					e.stopPropagation();
+																					HostApi.instance.send(OpenUrlRequestType, {
+																						url: newRelicUrl || " "
```
Comment (bcanzanella) [inline shared/ui/Stream/Observability.tsx:783]: if there's no `newRelicUrl` perhaps this external link icon should be hidden?

**Review by bcanzanella — CHANGES_REQUESTED** [review]
looks good, couple small comments.

**Comment by eric-j** [comment]
@bcanzanella I updated those requested changes

File: shared/ui/Stream/Observability.tsx [hunk 3]
```diff
@@ -394,6 +419,23 @@ export const Observability = React.memo((props: Props) => {
 		}
 	}, [derivedState.hiddenPaneNodes]);
 
+	useInterval(() => {
+		fetchGoldenMetrics(expandedEntity);
+	}, 60000);
```
[the remark this answers is not in the document]
Comment (bcanzanella) [reply shared/ui/Stream/Observability.tsx:424]: also, i don't believe i saw anything below, but i think working in some kind of lastUpdated value would be useful for the end users. perhaps it's in a tooltip or somewhere out of the way? That would let the user know that we're getting fresh stats. thoughts?

**Pull request merged**